style.css

```diff
@@ -82,3 +87,144 @@ body {
   display: inline;
   margin-right: 13px;
 }
+
+.container {
+  margin: 0 auto;
+  width: 85%;
+}
```

Comment:
- [ ] Nice job styling the projects, they look amazing :heart_eyes: But please in order to lay out the works use CSS Grid (`display: grid`) as required in the project requirement.

![Screenshot from 2023-03-24 16-24-46](https://user-images.githubusercontent.com/91142432/227584050-6fd500b6-4ae2-4971-a6c0-0930f2ce3f05.png)